<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>781</width>
    <height>736</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>참여형 교육 솔루션</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../gui/Solution/logo.png</normaloff>../gui/Solution/logo.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">#MainWindow {
background-color:url(:/images/bg.jpg)
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>781</width>
      <height>741</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">* {
font-family:century gothic;
font-size: 24px;
}


#login {
border-image:url(:/images/bg.jpg);
}

QPushButton
{
background:rgb(184, 244, 255);
border-radius:60px;
}

QPushButton:hover
{
color:#333;
border-radius:15px;
background:#49ebff;
}

#after_log_page {
border-image:url(:/images/bg.jpg);
}
</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="login">
     <widget class="QGroupBox" name="signin">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>250</y>
        <width>371</width>
        <height>271</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>century gothic</family>
        <pointsize>-1</pointsize>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="title">
       <string>Log In</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QLabel" name="label_id_2">
           <property name="font">
            <font>
             <family>century gothic</family>
             <pointsize>-1</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Student ID</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_id_2">
           <property name="font">
            <font>
             <family>century gothic</family>
             <pointsize>-1</pointsize>
            </font>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <widget class="QLabel" name="label_pw_2">
           <property name="font">
            <font>
             <family>century gothic</family>
             <pointsize>-1</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Password</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_pw_2">
           <property name="font">
            <font>
             <family>century gothic</family>
             <pointsize>-1</pointsize>
            </font>
           </property>
           <property name="echoMode">
            <enum>QLineEdit::Password</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="Button_login_2">
         <property name="font">
          <font>
           <family>century gothic</family>
           <pointsize>-1</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Log In</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="Button_register">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>680</y>
        <width>181</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>century gothic</family>
        <pointsize>-1</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Make Account</string>
      </property>
     </widget>
     <widget class="QWidget" name="topPanel" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>781</width>
        <height>101</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#topPanel {
background-color:rgb(158, 218, 255)
}</string>
      </property>
      <widget class="QLabel" name="label_title">
       <property name="geometry">
        <rect>
         <x>320</x>
         <y>20</y>
         <width>131</width>
         <height>61</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>century gothic</family>
         <pointsize>-1</pointsize>
        </font>
       </property>
       <property name="text">
        <string>임시 제목</string>
       </property>
      </widget>
      <widget class="QWidget" name="widget_logo" native="true">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>0</y>
         <width>181</width>
         <height>111</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">#widget_logo {
border-image:url(:/images/logo.png);
}</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="after_log_page">
     <widget class="QPushButton" name="Button_makeRoom">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>670</y>
        <width>161</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Make Room</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Button_toHome">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>670</y>
        <width>171</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Log Out</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_makeRoom">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>70</y>
        <width>671</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page"/>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
